fanuc_m10ia_support/launch/test_m10ia7l.launch

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<launch>
+  <include file="$(find fanuc_m10ia_support)/launch/load_m10ia7l.launch" />
+  <param name="use_gui" value="true" />
```

Comment:
>  I was just following the structure of the existing m10ia files.

I know, and that was fine, but the convention of making it a private parameter was introduced after the M-10iA support package was created. I haven't had an opportunity to fix everything up.